mmv1/templates/terraform/examples/apigee_organization_keyvaluemaps_beta_test.tf.tmpl

```diff
@@ -0,0 +1,148 @@
+resource "google_project" "project" { 
```

Comment:
From what I can tell, this beta-only test is not doing anything interesting vs the non-beta test. I would just remove both `_beta_test` tests.